site-popup/template/src/dashboard/hooks/useDashboardClient.tsx

```diff
@@ -0,0 +1,13 @@
+import { dashboard } from "@wix/dashboard";
+import { createClient } from '@wix/sdk';
+
+export const useDashboardSdk = () => {
```

Comment:
We are missing inferred types from the SDK maybe we can help here and cast to the dashboard SDK type so `observeState` not be any

```suggestion
import { SDK, dashboard } from "@wix/dashboard";
import { createClient } from '@wix/sdk';

export const useDashboardSdk = (): SDK => {
```

Reply:
I accepted the proposal but It is not `any`, it is typed even without this, you are correct that it is needed if I was using `@wix/dashboard-react` but I am not, because it does not support `observeState`, so I had to use the base one